ppl/src/main/antlr/OpenSearchPPLLexer.g4

```diff
@@ -52,12 +52,6 @@ WITH:                               'WITH';
 // CLAUSE KEYWORDS
 SORTBY:                             'SORTBY';
 
-// FIELD KEYWORDS
-AUTO:                               'AUTO';
-STR:                                'STR';
-IP:                                 'IP';
-NUM:                                'NUM';
```

Comment:
revert?

Reply:
please add any information about this non-functioning grammar to https://github.com/opensearch-project/sql/issues/3180
We should fix this issue instead of removing the grammar

Reply:
Reverted and added a `TODO`. `IP` keyword has been moved to the "data type" group - seems to me like it makes the most sense there 🤷